source/particle/output/ascii.cc

```diff
@@ -58,36 +64,26 @@ namespace aspect
           {
             // If it's a 1D element, print just the name, otherwise use []
             if (property->second == 1)
-              {
-                output << property->first << " ";
-              }
+              output << property->first << ' ';
             else
-              {
-                for (unsigned int i=0; i<property->second; ++i) output << property->first << "[" << i << "] ";
-              }
+              for (unsigned int component_index=0; component_index<property->second; ++component_index)
+                output << property->first << "[" << component_index << "] ";
           }
-        output << "\n";
+        output << '\n';
```

Comment:
Yes, \n is a single character (```0xa```) that you can write as ```'\n'```. It will be translated to whatever byte-sequence the system uses (```\r\n``` on windows).